src/FluentCMS.Web.UI/Components/Enums/InputTypeEnum.cs

```diff
@@ -0,0 +1,9 @@
+namespace FluentCMS.Web.UI.Components;
+
+public enum InputTypeEnum
+{
+    Date,
+    Datetime_Local,
```

Comment:
Dont use `_` to Seperate Words Just use PascalCase and `FromPascalCaseToKebabCase()` will handle the rest